content/applications/productivity/discuss.rst

```diff
@@ -1,105 +1,162 @@
+To delete direct message conversations, click the :guilabel:`X` next to the name in the
+:guilabel:`Direct Messages` section of the sidebar.
+
+.. note::
+  This does **not** delete the :guilabel:`Direct Messages` in the conversation. The direct message
+  conversation's history is visible when another direct message with the same person, or group, is
+  created.
+
+.. _discuss/meetings:
+
+Meetings
+========
+
+In **Discuss**, :guilabel:`Meetings` are video calls. To start a meeting from the **Discuss**
+application home page, click :guilabel:`Start a meeting` in the top-left corner, and select who to
+invite to the meeting, via the :guilabel:`Invite People` drop-down window that appears. To start a
+meeting from a direct message, click the :icon:`fa-phone` :guilabel:`Start a Call` icon in the
+top-right.
+
+.. image:: discuss/meeting.png
+   :alt: View of a Meeting in Odoo Discuss.
+
+Once a meeting has been started, the buttons on the bottom of the call screen can be used to
+:icon:`fa-microphone` :guilabel:`Mute/Unmute`, :icon:`fa-headphones` :guilabel:`Deafen/Undeafen`,
+and :icon:`fa-video-camera` :guilabel:`Turn camera on/off`. Clicking the :icon:`fa-ellipsis-v`
+:guilabel:`More` icon allows the user to :icon:`fa-hand-paper-o` :guilabel:`Raise Hand`,
+:icon:`fa-desktop` :guilabel:`Share Screen`, or :icon:`fa-arrows-alt` :guilabel:`Enter Full Screen`.
+
+Configuration
+=============
+
+Selecting :guilabel:`Configuration` at the top of the Discuss homepage triggers a drop-down menu of
+options including :guilabel:`Notification` :guilabel:`Voice & Video`, and :guilabel:`Canned
+Responses`.
+
+Select :menuselection:`Configuration --> Notification` to mute a server or adjust the channel's
+notification settings.
+
+Select :menuselection:`Configuration --> Voice & Video` to select a voice input device, select
+:guilabel:`Voice Detection` or :guilabel:`Push to Talk`, set the voice detection threshold, and
+toggle :guilabel:`Show video participants only` and :guilabel:`Blur video background`.
+
+Select :menuselection:`Configuration --> Canned Responses` to configure canned responses.
+
+.. _discuss_app/notification_preferences:
+
+Choose notifications preference
+===============================
+
+Access user-specific preferences for the *Discuss* app by navigating to :menuselection:`Settings app
+--> Manage Users`, select a user, and then click the :menuselection:`Preferences` tab.
+
+.. image:: discuss/preferences-user.png
+   :alt: View of the Preferences tab for Odoo Discuss.
+
+By default, the :guilabel:`Notification` field is set as :guilabel:`Handle by Emails`. With this
+setting enabled, a notification email is sent by Odoo every time a message is sent from the
+chatter, a note is sent with an `@` mention (from *Chatter*), or a notification is sent for a record
+that the user follows. Something that triggers a notification is changing of the stage (if an email
+is configured to be sent, for example if the task is set to :guilabel:`Done`).
```

Comment:
Can you please simplify the wording of this last sentence to enhance its readability, please? Thanks!